### DISCUSSION TITLE
For Arch Linux Users

### DISCUSSION THREAD

**Discussion by axiomgraph** [friction2d/friction#60] [General]
If you are an Arch Linux user and want to use Friction, you can use Archalien.

https://github.com/Asher256/archalien

This program will convert the Debian package to the Arch package.
Here is an example command:
`
./archalien.py '/home/tiger/archalien/friction-0.9.3-ubuntu22.04.deb'`

which produces
friction-0.9.3-1-x86_64.pkg.tar.gz
then use `sudo pacman -U '/home/tiger/archalien/friction-0.9.3-1-x86_64.pkg.tar.gz' `

**Comment by axiomgraph** [comment]
oops. not a proper solution  context menu does not work.

**Comment by rodlie** [comment]
Don't use packages made for other distros, that will never work properly.

**Comment by rodlie** [comment]
Btw, I can add archlinux to "supported" distros, this means I provide binary packages on releases.

The only problem with rolling-release distros is that the package I provide will break sometime in the future. And could introduce build issues right before a release.

Probably best solved in AUR.

**Comment by axiomgraph** [comment]
Well, I don't think that would be a good idea at this point. I was just checking if archalien works.

**Comment by rodlie** [comment]
Ok.

The latest build instructions for Linux can be found here: https://friction.graphics/documentation/source-linux.html
